pkgs/development/interpreters/python/fetchpypi.nix

```diff
@@ -20,7 +20,7 @@ let
 
   in compute (builtins.removeAttrs attrs ["format"]);
 
-in makeOverridable( {format ? "setuptools", sha256 ? "", hash ? "", ... } @attrs:
```

Comment:
This might be a false positive `builtins.functionArgs`, which `makeOverridable` uses will return a different result.